pcsx2/GS/Renderers/Common/GSDevice.h

```diff
@@ -21,6 +21,11 @@
 #include "GSVertex.h"
 #include "GS/GSAlignedClass.h"
 #include "GSOsdManager.h"
+#include <array>
+#ifdef _WIN32
+#include <dxgi1_3.h>
+#endif
```

Comment:
there some reason you stuck this here instead of device11?

Reply:
No probably not, I think I just slapped includes around until it built.

Can move it.